### PR TITLE
Fix bug in utils/url isValidUrl and use Bugsnag.notify instead of console.error logging

### PR DESCRIPTION
**Pull request by 0xdevalias**
Using `Bugsnag.notify` will give us visibility into issues that we would otherwise never see if they are only logged with `console.error`

### CODE REVIEW

File: src/utils/url.ts [hunk 1]
```diff
@@ -40,15 +41,25 @@ export const openRoomUrl = (url: string) => {
...
+
+  window.open(url, "_blank", "noopener,noreferrer");
 };
 
-export const isValidUrl = (url: string) => {
-  return VALID_URL_PROTOCOLS.includes(new URL(url).protocol);
+export const isValidUrl = (url: string): boolean => {
+  try {
+    return VALID_URL_PROTOCOLS.includes(new URL(url).protocol);
+  } catch (e) {
```
Comment (thecadams) [inline src/utils/url.ts:60]: nit: consider catching only `TypeError` here: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

**Review by thecadams — APPROVED** [review]
Looks good, only one nit, beginnings of observability greatness here.

File: src/utils/url.ts (hunk 1 above)
Comment (0xdevalias) [reply src/utils/url.ts:60]: resolved in https://github.com/sparkletown/sparkle/pull/889/commits/935aeb69024ccebf57b50e64566a4edd511425d7

**Comment by thecadams** [comment]
[Bring Back Badges](https://trello.com/c/AAmGbZVs/447-bring-back-badges)

**Pull request merged**